command/agent/job_endpoint_test.go

```diff
@@ -1981,29 +1983,150 @@ func TestHTTP_JobDispatch(t *testing.T) {
 
 		// Make the HTTP request
 		req2, err := http.NewRequest(http.MethodPut, "/v1/job/"+job.ID+"/dispatch", buf)
+		must.NoError(t, err)
 		if err != nil {
-			t.Fatalf("err: %v", err)
+			t.Logf("err: %v", err)
 		}
 		respW.Flush()
 
 		// Make the request
 		obj, err := s.Server.JobSpecificRequest(respW, req2)
 		if err != nil {
-			t.Fatalf("err: %v", err)
+			t.Logf("err: %v", err)
```

Comment:
This no longer exits on error so we've broken the assertion here. Looks like this block could be turned into `must.NoError(t, err)` instead.